src/main/resources/static/css/style2.css

```
@@ -274,3 +280,156 @@ main{
+}
+
+.user-name {
+    font-size: 16px;
+    font-weight: 600;
+    white-space: nowrap;
+}
+
+.user-email {
+    font-size: 14px;
+    color: #666;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)